src/templates/synonyms.tsv

```diff
@@ -1,4 +1,3 @@
 Ontology ID	Label	Exact synonym	Close Synonym	Related synonym	Broad synonym	Narrow synonym	Contributor
 ID		A oboInOwl:hasExactSynonym SPLIT=|	A oboInOwl:hasCloseSynonym SPLIT=|	A oboInOwl:hasRelatedSynonym SPLIT=|	A oboInOwl:hasBroadSynonym SPLIT=|	A oboInOwl:hasNarrowSynonym SPLIT=|	AI dc:contributor SPLIT=|
-http://purl.obolibrary.org/obo/OBA_VT0010023	forelimb length			forearm length			https://orcid.org/0000-0003-2699-3066

```

Comment:
I dont know what happened here but this pull request has nothing to do with the Title of the pull request. Can you double check this?

Reply:
I think I added it originally but then removed it because we decided that one can get to forearm from forelimb if I remember correctly.

Reply:
Thats ok, but how does the title of this pull request relate to the change here? The title suggests something is added in this PR!

Reply:
oh, perhaps I'm confused. I re-ran it, and committed the changes. Did I not add "chest girth"?

Reply:
No! Have you double-checked the Google sheets, that the chest girth synonym is present there? Which commands are you running? Perhaps you didn't run the google sheets sync command?

Reply:
I ran:

`
cd src/ontology
sh run.sh make sync_templates_google_sheets
sh run.sh make recreate-synonyms
`

I removed the forelimb synonym, which is maybe why it is showing up as removed?

Reply:
Everything is correct then. Thanks for checking! I also double checked now and your PR indeed reflects the state of the Google Docs